readthedocs/projects/management/commands/import_project_from_live.py

```diff
@@ -1,72 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I still use this helper. I think it should be pretty simple to adopt it to v2.

Reply:
ha, I use it too :(

Reply:
Yes. This command is useful when debugging production projects locally.

I'd like to find a way to keep it.

Reply:
I'll update this with an adapted version using the v2 API.

Reply:
We can do it in another PR, tho :)

Reply:
Nah, I think it isn't too onerous... Also, that way the single PR has the full set of changes.

Reply:
I re-added this although there are some changes. The biggest one is that in order to get all the fields from an API response, the request needs to be authenticated and the user must be staff ([See this](https://github.com/rtfd/readthedocs.org/blob/3218c808e87206983c4eb3ce4faceddbcc0f7aa3/readthedocs/restapi/views/model_views.py#L82-L95)). One key field that is only returned with the `ProjectAdminSerializer` is the `requirements_file`.

By default, the management command will be unauthenticated. However, if you have `READ_THE_DOCS_USERNAME` and `READ_THE_DOCS_PASSWORD` in your environment variables when you execute the management command, they'll be used.